spec/controllers/admin/conferences_controller_spec.rb

```diff
@@ -280,6 +292,18 @@
         end
       end
     end
+
+    describe 'GET #conference_wide_screen' do
+      it 'requires organizer privileges' do
```

Comment:
This belongs to the ability test, doesn't it?

https://github.com/openSUSE/osem/blob/master/spec/models/ability_spec.rb